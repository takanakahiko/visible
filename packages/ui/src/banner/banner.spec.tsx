import React from 'react';
import { render } from '@testing-library/react';
import { Banner } from '.';

describe('Banner', () => {
  it('matches snapshot', () => {
    const { container } = render(<Banner />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
