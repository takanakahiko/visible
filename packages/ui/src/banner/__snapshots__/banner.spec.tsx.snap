// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner matches snapshot 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  width: 1080px;
  margin: auto;
  color: white;
}

.c0 {
  background-color: #ffaa01;
  box-shadow: 0 0 3px rgba(255,170,1,0.84);
}

<header
  class="c0"
>
  <div
    class="c1"
  >
    Visible
  </div>
</header>
`;
